scripts/dev

```diff
@@ -42,12 +42,25 @@ needs "gotest", install_with: "go get -u github.com/rakyll/gotest"
 needs "goimports", install_with: "go get -u golang.org/x/tools/cmd/goimports"
 needs "ulimit"
 
+# Gracefully handle folks trying to explore using common help flags
```

Comment:
I watched someone try to run `scripts/dev --help` and they got the internal Rake help instead of anything useful.